tests/ConnectionTest.php

```diff
@@ -413,6 +414,35 @@ public function testCommitStartsTransactionInNoAutoCommitMode(): void
         self::assertTrue($conn->isTransactionActive());
     }
 
+    public function testBeginTransactionFailureAfterCommitInNoAutoCommitMode(): void
+    {
+        $driverConnectionMock = $this->createMock(DriverConnection::class);
+        $driverConnectionMock->expects(self::exactly(2))
+            ->method('beginTransaction')
+            ->willReturnOnConsecutiveCalls(
+                true,
+                self::throwException(new RuntimeException()),
+            );
+
+        $driverMock = $this->createStub(Driver::class);
```

Comment:
```suggestion
        $driver = $this->createStub(Driver::class);
```

Reply:
the name? The whole file uses drivermock 🤔

Reply:
None of the variables in that file that are filled with `createStub` are suffixed though, right?

Reply:
ah got it